src/plugins/data/public/query/query_string/language_service/types.ts

```diff
@@ -35,6 +35,11 @@ export interface EditorEnhancements {
   queryEditorExtension?: QueryEditorExtensionConfig;
 }
 
+export interface SampleQuery {
```

Comment:
i think we should create an action item to unify this and recent query  and saved queries to extend from a base class. because i can see a world where we add buttons near the sample queries and instead of fighting to update to set the query behavior subject we can just push the entire query object. for example, it will set the date range to 1 week before and 1 week after.